### DISCUSSION TITLE
Stateless replication from Postgres and MySQL to Snowflake

### DISCUSSION THREAD

**Discussion by abcpro1** [getdozer/dozer#2267] [General]
The design notes are in this URL
https://www.tldraw.com/r/3iMbOi3USiSacd1OeHtYo?viewport=149,-62,1728,992&page=page:page

**Comment by chubei** [comment]
1. I was under the impression that postgres cannot resume in the middle of snapshotting. @karolisg can you clarify?
2. The purpose of this design is to skip using `dozer-log` for data movement. Is that right?

**Reply by abcpro1** [reply]
2. Right. The goal is to increase the throughput and lower the cost of data replication.

**Reply by karolisg** [reply]
1. We cannot resume from the middle of snapshotting. To have this we need to implement something like this
https://debezium.io/blog/2021/10/07/incremental-snapshots/
